<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAChasySettingsForm</class>
 <widget class="QWidget" name="SAChasySettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="mPravSutKrugCheckBox">
     <property name="text">
      <string>Правильный суточный круг</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mShowDolyCheckBox">
     <property name="text">
      <string>Показать стрелку доль</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mIgnoreDSTCheckBox">
     <property name="text">
      <string>Не выполнять переход на летнее время</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mSimpleCheckBox">
     <property name="text">
      <string>Упрощённый вид</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mShowDateCheckBox">
     <property name="text">
      <string>Показать дату</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Фон:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="SAColorButton" name="mSAChasyBgPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Интервал обновления:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="mUpdateIntervalSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>45</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ширина и высота:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="mSAChasySizeSpinBox">
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SAColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">sacolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
